Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -805,6 +805,13 @@ theorem prod_eq_single {s : Finset α} {f : α → β} (a : α) (h₀ : ∀ b 
 #align finset.prod_eq_single Finset.prod_eq_single
 #align finset.sum_eq_single Finset.sum_eq_single
 
+@[to_additive]
+lemma prod_univ_eq_single [Fintype α] {f : α → β} (a : α)
```

Comment:
I think we might already have this in the `Fintype` namespace.

Reply:
I was very surprised but couldn't find it. But I just searched again after `import Mathlib` and indeed it exists.